zerver/actions/message_edit.py

```diff
@@ -812,16 +812,19 @@ def user_info(um: UserMessage) -> Dict[str, Any]:
     send_event(user_profile.realm, event, users_to_be_notified)
 
     sent_resolve_topic_notification = False
-    if (
-        topic_name is not None
-        and new_stream is None
-        and content is None
-        and len(changed_messages) > 0
-    ):
-        assert stream_being_edited is not None
+    if topic_name is not None and content is None and len(changed_messages) > 0:
+        # When stream is changed and topic is marked as resolved|unresolved
```

Comment:
It is better to use "or" instead of "|" in the comments. Also we should use double quotes `"` instead of single quotes below just for consistency.